src/main/java/org/jabref/gui/groups/GroupNodeViewModel.java

```diff
@@ -250,13 +253,19 @@ private void updateMatchedEntries() {
         // We calculate the new hit value
         // We could be more intelligent and try to figure out the new number of hits based on the entry change
         // for example, a previously matched entry gets removed -> hits = hits - 1
-        BackgroundTask
-                .wrap(() -> groupNode.findMatches(databaseContext.getDatabase()))
-                .onSuccess(entries -> {
-                    matchedEntries.clear();
-                    matchedEntries.addAll(entries);
-                })
-                .executeWith(taskExecutor);
+        if (preferencesService.getDisplayGroupCount()) {
+            BackgroundTask
+                    .wrap(() -> groupNode.calculateNumberOfMatches(databaseContext.getDatabase()))
```

Comment:
I think this duplication of the code is a result from a merge conflict?

Reply:
Looks like yes, this is a conflict.
the second `BackgroundTask` is not mine. My code is:
```
    private void calculateNumberOfMatches() {
        // We calculate the new hit value
        // We could be more intelligent and try to figure out the new number of hits based on the entry change
        // for example, a previously matched entry gets removed -> hits = hits - 1
        if (preferencesService.getDisplayGroupCount()) {
            BackgroundTask
                    .wrap(() -> groupNode.calculateNumberOfMatches(databaseContext.getDatabase()))
                    .onSuccess(hits::setValue)
                    .executeWith(taskExecutor);
        }
    }
```

Reply:
Sorry, I haven't worked with conflicts yet.
Looks like this is your correction as of 11.04.2020. Can I do something to resolve this GIT conflict?

Reply:
No problem! I also often have problems with git ;-)

Here the solution is relatively simple. Just edit the code and remove the part that is duplicated / not correct. Then commit and push.